modules/402-ingress-nginx/images/protobuf-exporter/pkg/server/telemetry.go

```diff
@@ -144,6 +174,108 @@ func (s *TelemetryServer) handleConn(c *net.TCPConn) {
 	}
 }
 
+func (s *TelemetryServer) isResourceExcluded(namespacedIngress string) bool {
```

Comment:
There is no resources in protobuf exporter. Neither ingresses, nor namespaces.
I think there should be the way to make it generic.

Proposal: add annotations.
```
map<string, string> Annotations = 4;
```
Send ns and ingress as annotations.

Then, add a the way to filter metrics by annotations.

P.S. There is a config file with mappings, maybe it is possible to reuse mappings somehow https://github.com/deckhouse/deckhouse/blob/main/modules/402-ingress-nginx/images/protobuf-exporter/rootfs/etc/protobuf_exporter/mappings.yaml